build/storage/scripts/disk_infrastructure.sh

```diff
@@ -19,30 +19,30 @@ EOF
 
 function is_virtio_blk_attached() {
     return $(python3 <<- EOF
-from scripts import disk_infrastructure
+from scripts.disk_infrastructure import is_virtio_blk_attached
 
-print(disk_infrastructure.is_virtio_blk_attached(sock="${1}"))
+print(0 if is_virtio_blk_attached(sock="${1}") else 1)
```

Comment:
wouldn't int(is_virtio_blk_attached(sock="${1}")) do?  why do you return 0 if the case is true?

Reply:
It needs to be a return code for bash function so 0 means that it went well, and 1 that there was an error.

Reply:
I would still suggest avoiding doing this - see https://stackoverflow.com/questions/5431909/returning-a-boolean-from-a-bash-function/43840545#43840545

Reply:
I've changed it. Now it is connected to exit status of Python function.